build/build-report.js

```diff
@@ -110,19 +115,27 @@ async function buildUmdBundle() {
   const bundle = await rollup.rollup({
     input: 'report/clients/bundle.js',
     plugins: [
+      rollupPlugins.inlineFs({verbose: true}),
       rollupPlugins.commonjs(),
       rollupPlugins.terser({
         format: {
           beautify: true,
         },
       }),
+      // Shim this empty to ensure the bundle isn't 10MB
```

Comment:
maybe we should have a test for this

Reply:
> maybe we should have a test for this

![fry squinting](https://user-images.githubusercontent.com/316891/166512742-23cf5a15-a42a-4c7e-a87e-0ffb36e69b0e.png)